tests/escrow/programs/escrow/src/lib.rs

```diff
@@ -102,7 +102,7 @@ pub mod escrow {
 #[derive(Accounts)]
 #[instruction(initializer_amount: u64)]
 pub struct InitializeEscrow<'info> {
-    #[account(signer)]
+    #[account(signer, mut)]
     pub initializer: AccountInfo<'info>,
```

Comment:
`Signer`

Reply:
@paul-schaaf the reason why I left these untouched is because they are used in CPI calls which have a struct defined in other parts of the framework, like here `anchor_spl::token::SetAuthority`, or below `anchor_spl::dex::OrderbookClient`. What do you reckon should be done there?

Reply:
to be a bit clearer, the issue it that those CPI structs expect an `AccountInfo`, not a `Signer`

Reply:
you can use `to_account_info()` on signer